_translations/en.yml

```diff
@@ -676,9 +676,12 @@ en:
     anonymous: "Bitcoin is not anonymous"
     anonymoustxt: "Some effort is required to protect your privacy with Bitcoin. All Bitcoin transactions are stored publicly and permanently on the network, which means anyone can see the balance and transactions of any Bitcoin address. However, the identity of the user behind an address remains unknown until information is revealed during a purchase or in other circumstances. This is one reason why Bitcoin addresses should only be used once. Always remember that it is your responsibility to adopt good practices in order to protect your privacy. <a href=\"#protect-your-privacy#\"><b>Read more about protecting your privacy</b></a>."
     instant: "Unconfirmed transactions aren't secure"
-    instanttxt: "A Bitcoin transaction is usually deployed within a few seconds and begins to be confirmed in the following 10 minutes. During that time, a transaction can be considered authentic but still reversible. Dishonest users could try to cheat, which means there is a risk when accepting unconfirmated transactions. For large amounts like 1000 US$, it makes sense to wait for 6 confirmations or more. Each confirmation <i>exponentially</i> decreases the risk of a reversed transaction."
+    instanttxt: "New transactions start out as unconfirmed. If you trust the person paying you, you don't have to worry about unconfirmed transactions. If you don't trust them, you should wait for the transaction to receive at least one confirmation—this has historically provided a better than 95% guarantee that the payment can't be changed. Because Bitcoin miners sometimes misbehave, this guarantee isn't as strong for lightweight wallets, so you may want to wait for three confirmations unless you use a recent version of <a href=\"#download#\">Bitcoin Core</a>. For high value amounts, waiting for six or more confirmations is strongly recommended no matter what wallet you use."
```

Comment:
@harding That makes a lot of sense, that's clearer (and shorter, actually. Taking space on the page isn't really a problem)

Maybe it's possible to mention the 10 minute delay and the nature of the risk in the short paragraph? The example below fits on two lines.

~~~
New transactions start out as unconfirmed and only become irreversible after receiving 
enough confirmations. Each confirmation takes 10 minutes on average, and how many
you should wait for depends on your situation:
~~~